queue_services/entity-filer/src/entity_filer/common/enum.py

```diff
@@ -37,6 +38,7 @@ def get_enum_by_name(cls, value: str) -> Optional[str]:
                 return enum_value
         return None
 
+    # pylint: disable-next=no-self-argument
     def _generate_next_value_(name, start, count, last_values):
```

Comment:
Not entirely sure why the `self` argument is needed here. If it is okay to pass `self` to this function, I will add it in, but for now I have ignored the error.